### ISSUE TITLE
[BUG] All data fetching is failing [ExchangeRateDataService] No exchange rate has been found for USDEUR

### ISSUE THREAD

**Issue by Timbo925** [ghostfolio/ghostfolio#2470]
**Bug Description**

I'v encountered a weird problem where suddenly after an update all my data is not loading anymore in the UI. I can still see my activities but no details or portfolio. 

**Screenshots**
<img width="1446" alt="Screenshot 2023-10-11 at 11 51 02" src="https://github.com/ghostfolio/ghostfolio/assets/4271128/d3e88cfb-36b5-4f99-bb72-ac363e5e4686">

<img width="479" alt="image" src="https://github.com/ghostfolio/ghostfolio/assets/4271128/990c2640-95b1-41f5-b9f8-adb07a0a0bf7">

<img width="1086" alt="Screenshot 2023-10-11 at 12 06 26" src="https://github.com/ghostfolio/ghostfolio/assets/4271128/40490364-ff3f-49d1-b1f0-c64360b3b7fe">

**Logs**

I see mainly the following errors in the logs. I've used the market data page already to refetch the historical data and this seems to work ok. Still getting the error afterwards. Historical data is also fetching correctly for the USDEUR pair, have green boxes (see screenshots).

Base Currency is set to EUR, seems to be still using USD in the 3rd screenshot, maybe this is a problem? 

```
[Nest] 230  - 10/11/2023, 2:49:29 AM   ERROR [ExchangeRateDataService] No exchange rate has been found for USDEUR
[Nest] 230  - 10/11/2023, 2:49:29 AM    WARN [PortfolioCalculator] Missing historical market data for ABI.BR (YAHOO)

Error while opening activity details.

[Nest] 229  - 10/11/2023, 3:15:13 AM   ERROR [ExceptionsHandler] Cannot read properties of undefined (reading 'items')
TypeError: Cannot read properties of undefined (reading 'items')
    at PortfolioService.getPosition (/ghostfolio/apps/api/main.js:1:324181)
    at async PortfolioController.getPosition (/ghostfolio/apps/api/main.js:1:345262)
```

**Environment**

-  Self-hosted - Docker
- Ghostfolio Version v2.10.0
- Browser - Arc/Chrome
- OS - Mac

**Additional context**

Any help narrowing down the problem highly appreciated.

**Comment by Timbo925** [comment]
Partial fix has been found, I added the TZ variable to the docker startup command and left it clear. This resolved most of my problems. The summary page is still erroring out tho.

![image](https://github.com/ghostfolio/ghostfolio/assets/4271128/9c698368-4556-4527-b026-491b5df48fe9)

````
[Nest] 230  - 10/12/2023, 5:46:15 PM   ERROR [ExceptionsHandler] 
Invalid `prisma.account.findMany()` invocation:

{
  cursor: undefined,
  include: {
    Platform: true,
    balances: {
      orderBy: {
        date: "desc"
      },
      take: 1
    }
  },
  orderBy: undefined,
  skip: undefined,
  take: undefined,
  where: {
    id: {
      in: [
        "5e9a86a2-8f04-4d8b-9223-4f2417ae89af",
        "46cf5168-5bb3-4a42-b43a-d7e28b87945a",
        null,
        "e0df5619-9770-430b-8b1d-848bb5f41d48"
      ]
      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    }
  }
}

Argument `in`: Invalid value provided. Expected ListStringFieldRefInput, provided (String, String, Null, String).
````
Clearly `null`  is the problem here and the UUID is referencing the Accounts.  Looking at the DB I only see 3 acocunts tho.
**Any idea where this list of accounts is fetched?**

![image](https://github.com/ghostfolio/ghostfolio/assets/4271128/6e52b2c8-eade-4209-8cdb-57c410e7bae2)

**Comment by Timbo925** [comment]
Seems to have fixed itself after the new recent update. Closing the issue.

**Comment by dtslvr** [comment]
> Seems to have fixed itself after the new recent update. Closing the issue.

Thank you for your feedback. It has probably been fixed via #2598.
